architecture-docs/adr-003.md

```diff
@@ -0,0 +1,144 @@
+# ADR 003: Migrate to jellyfish merkle trie
+
+## Changelog
+* 31-03-2020: Initial Draft
```

Comment:
there were changes from the initial write up?

Reply:
I think no need to track changed not yet committed?